user-key-value-api-spec.yaml

```diff
@@ -0,0 +1,193 @@
+openapi: 3.0.0
+info:
+  description: 'The HuBMAP User Key/Value API is a RESTful web service with create/update, read, and delete operations for key/value pairs associated with an authorized user, which persist in a backend data store.'
+  version: 1.0.0
+  title: HuBMAP UKV API
+  contact:
+    name: HuBMAP Help Desk
+    email: help@hubmapconsortium.org
+  termsOfService: 'https://hubmapconsortium.org'
```

Comment:
I don't know if we have a formal terms of service, better not to add it for now

Reply:
Copied that [from search-api](https://github.com/hubmapconsortium/search-api/blob/55930a168e8e8f068f784a8791b49523d027bbef/search-api-spec.yaml#L6). Removed.